cabal.project

```diff
@@ -37,8 +37,8 @@ extra-packages: ieee, filemanip
 -- when decoding invalid (e.g. malicious) text literals.
 source-repository-package
   type: git
-  location: https://github.com/Quid2/flat.git
-  tag: ee59880f47ab835dbd73bea0847dab7869fc20d8
+  location: https://github.com/Quid2/flat
+  tag: 44dd79ab3d6547afd7674876f0307995fc40dfcf
```

Comment:
what does this do?

Reply:
I don't know the story behind it, but this is the `HEAD` of our fork here: https://github.com/input-output-hk/flat

The fork has been pushed upstream.